src/main/resources/application.properties

```diff
@@ -98,6 +98,8 @@ rundeck.project.group.quickstarter=quickstarts
 rundeck.project.group.openshift=openshift
 rundeck.project.openshift.create.name=create-projects
 
+jenkinspipeline.quickstarter={beSpringBoot: 'http://192.168.56.31:7990/scm/prov/quickstarter-jenkins.git#master/beSpringBoot/Jenkinsfile'}
+jenkinspipeline.create-ods-projects-job=http://192.168.56.31:7990/scm/opendevstack/ods-core.git#production/create-projects/Jenkinsfile
```

Comment:
see above

Reply:
Yes, absolutely! This was just temporary so that we can discuss the current state. @kiwo I think it would be good to not use an absolute URI path at all but to narrow the "scope" to a repository underneath `opendevstack` ... so either users pick e.g. `ods-core` or `ods-quickstarters` ... or their own `my-super-cool-quickstarters`, but for security reasons (and access rights) it is probably good to restrict to `opendevstack` project on your BitBucket instance.

Reply:
Depends when you can change the quickstarter URL. If you could specifiy alternative repositories at runtime, then I would completely agree on your comment. If it is something that is once set at deployment time, we could leave the complete URL in.

Reply:
@rattermeyer Not quite, because it affects the webhook proxy. If you have full URLs that you send to the proxy/Jenkins than it is problematic if someone would be able to bypass security. better to just be able to build pre-configured projects. See https://github.com/opendevstack/ods-provisioning-app/issues/270.